tensorflow/python/keras/benchmarks/keras_examples_benchmark_test.py

```diff
@@ -0,0 +1,91 @@
+# Copyright 2020 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""Benchmark for examples on https://keras.io/examples."""
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import six
+
+import tensorflow as tf
+
+from tensorflow.python.keras.benchmarks import benchmark_util
+from tensorflow.python.platform import benchmark
+from tensorflow.python.platform import test
```

Comment:
I believe both of them are public APIs too: `tf.test.Benchmark` and `tf.test.TestCase`. Please update it.

Reply:
I can use `tf.test.Benchmark`, but for the `benchmark.ParameterizedBenchmark`, I didn't find the public APIs, so I keep the`from tensorflow.python.platform import benchmark`.

Reply:
Yeah, makes sense. Feel free to keep the import, while using as many public APIs as possible.